File: src/main/java/itmo/is/project/repository/module/dock/DockModuleBlueprintRepository.java
package itmo.is.project.repository.module.dock;

import itmo.is.project.model.module.dock.DockModuleBlueprint;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface DockModuleBlueprintRepository extends JpaRepository<DockModuleBlueprint, Integer> {
}

File: src/main/kotlin/itmo/isproject/repository/module/dock/DockModuleBlueprintRepository.kt
package itmo.isproject.repository.module.dock

import itmo.isproject.model.module.dock.DockModuleBlueprint
import itmo.isproject.repository.module.ModuleBlueprintRepository
import org.springframework.stereotype.Repository

@Repository
interface DockModuleBlueprintRepository : ModuleBlueprintRepository<DockModuleBlueprint>
